jsonnet/thanos-mixin/alerts/store.libsonnet

```diff
@@ -0,0 +1,88 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thanos-store.rules',
+        rules: [
+          {
+            alert: 'ThanosStoreGrpcErrorRate',
+            annotations: {
+              message: 'Thanos Store is returning Internal/Unavailable errors.',
+            },
+            expr: |||
+              rate(
+                grpc_server_handled_total{grpc_code=~"Unknown|ResourceExhausted|Internal|Unavailable", %(thanosStoreSelector)s}[5m]
+              ) > 0
```

Comment:
if this is a counter, once it increments once, it will be greater than 0 forever. let's make this greater than e.g. 5% of total requests